src/attributes.jl

```diff
@@ -1117,6 +1140,31 @@ function throw_set_error_fallback(::ModelLike, ::ConstraintSet,
     type $(typeof(set)). Use `transform` instead."""))
 end
 
+"""
+    ConflictParticipationStatusCode
+
+An Enum of possible values for the `ConstraintConflictStatus` attribute.
+This attribute is meant to indicate whether a given constraint participates
+or not in the last computed conflict.
+
+Possible values are:
+* `NOT_IN_CONFLICT`: the constraint does not participate in the conflict
+* `IN_CONFLICT`: the constraint participates in the conflict
+* `MAYBE_IN_CONFLICT`: the constraint may participate in the conflict,
+  the solver was not able to prove that the constraint can be excluded from
+  the conflict
+"""
+@enum(ConflictParticipationStatusCode, NOT_IN_CONFLICT, IN_CONFLICT, MAYBE_IN_CONFLICT)
+
+"""
+    ConstraintConflictStatus()
+
+A constraint attribute indicating whether the constraint participates
+in the conflict. Its type is `ConflictParticipationStatusCode`.
+Some solvers call the conflict IIS.
```

Comment:
This note is probably best put in `compute_conflict!` as a `Note: some solvers call a set of conflicting constraints an Irreducible Inconsistent Subsystem (IIS).`